### PR TITLE
handbrake: 1.3.0 -> 1.3.1

### PR DESCRIPTION
**Pull request by bbigras**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

/nix/store/j04a75bb47xq3zc43x78vkqbanw2l4ld-handbrake-1.3.0	 638.2M
/nix/store/12hv0y40c1fvx8gafh2h8l9kw22fla01-handbrake-1.3.1	 638.2M

**Note** The URL changed... Wouldn't it be better to fetch from github instead?

### CODE REVIEW

**Comment by jtojnar** [comment]
If I recall correctly, upstream complained about not using their provided tarball.

**Comment by bbigras** [comment]
Anything needed about https://github.com/bbigras/nixpkgs/blob/6443b0339e26d66010d8b5315af2f8ec7054b7cf/pkgs/applications/video/handbrake/default.nix#L105 ?

**Comment by wmertens** [comment]
Lgtm - @bbigras, try removing that flag to see if it is still needed?

**Comment by bbigras** [comment]
It builds and `nixos/tests/handbrake.nix` runs without the flag so I removed it.

**Comment by Anton-Latukha** [comment]
@bbigras Thank you.

@jtojnar Story was such: They have different tarballs on GitHub and on the official site, some files/data included only into official site tarball, so the mutual decision with them then was not to have the same tarball, but for Nix to fetch the tarball from the official site. If I remember correctly, they have their own internal elaborate versioning system, that also had been changed a couple of times, and that versioning system files are included into site tarball, but absent from GitHub tarball, so the compilation of the latter compilation fails with a version error.

**Comment by Anton-Latukha** [comment]
@jtojnar Story of this package was such:

TL;DR
They have different tarballs on GitHub and on the official site.

Some files/data included only into official site tarball, so the mutual decision with them then was not to have the same tarball, but for Nix to fetch the tarball from the official site. If I remember correctly, they have their own internal elaborate versioning system, that also had been changed a couple of times, and that versioning system files are included into site tarball, but absent from GitHub tarball, so the compilation of the latter compilation fails with a version error.

**Comment by bbigras** [comment]
Please review when you have time.

**Review by Anton-Latukha — APPROVED** [review]

**Review by JohnAZoidberg — APPROVED** [review]
Looks good. NixOS test runs through, GUI runs fine.

**Comment by bbigras** [comment]
Thanks!
